argus/environment.py

```diff
@@ -157,3 +217,22 @@ def _wait_for_api():
                 break
             except Exception:
                 time.sleep(1)
+
+    def _get_services(self):
+        services = self._run_command("systemctl -a")
+        services.extend(self._run_command("systemctl -a"))
+        services = [s for s in services if s.startswith('openstack') or s.startswith('neutron')]

```

Comment:
In fact, one solution would be to put the entire command in the environment's config:

systemctl -a|grep -oE "openstack|neutron"

Or something of sorts.